Source/UI/Screens/Error/ErrorViewController.swift

```diff
@@ -93,6 +93,7 @@ class ErrorViewController: IdentityUIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.sheetBackgroundView.layer.cornerRadius = self.theme.geometry.cornerRadius
```

Comment:
Could maybe be better to use a `UIBezierPath()` if you want smoother corners..
https://twitter.com/nathangitter/status/992842150500069376

Reply:
Nice, I switch to bezier everywhere but in some places it was much worse:

<img width="47" alt="image" src="https://user-images.githubusercontent.com/2638284/41155008-7850a8fe-6b1c-11e8-868c-2af261f6d8fb.png"> VS <img width="56" alt="image" src="https://user-images.githubusercontent.com/2638284/41155036-94febd56-6b1c-11e8-9442-3c6aadb8683a.png">

Becomes a bit more complicated to do if you want to add borders apparently.